BizHawk.Emulation.Cores/Consoles/Nintendo/GBA/VBANext.IStatable.cs

```diff
@@ -1,41 +1,11 @@
 ﻿using System;
 using System.IO;
-using Newtonsoft.Json;
-
 using BizHawk.Emulation.Common;
 
 namespace BizHawk.Emulation.Cores.Nintendo.GBA
 {
 	public partial class VBANext : IStatable
 	{
-		public bool BinarySaveStatesPreferred => true;
-
-		public void SaveStateText(TextWriter writer)
-		{
-			var s = new TextState<TextStateData>();
-			s.Prepare();
-			var ff = s.GetFunctionPointersSave();
-			LibVBANext.TxtStateSave(Core, ref ff);

```

Comment:
As others

Reply:
I'll assume ditto here, if not, oh well, we are a few mGBA memory hooks from deleting this core anyway